tools/wait_for_assisted_service.py

```diff
@@ -11,7 +11,7 @@
 SERVICE = "assisted-service"
 TIMEOUT = 60 * 30
 REQUEST_TIMEOUT = 2
-SLEEP = 3
+SLEEP = 10
```

Comment:
Why this sleep is even needed?
The major drawback of using sleeps is not only that we might waste time (not relevant in this case, setup time will be more), but also that it's a long-term maintenance
Is it possible to remove it and let the waiting-loop handle failures for the first couple of seconds?

Reply:
We are using the [waiting](https://pypi.org/project/waiting/) python package. By default, it runs every 1s and I think we can extend this interval to 10s at least. Otherwise, it just spams.

Reply:
ok, so not critical